### PR TITLE
feat(config): add a list of tools that support this config file format

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: scw/config.go [hunk 1]
```diff
@@ -22,6 +22,11 @@ const (
 const configFileTemplate = `# Scaleway configuration file
 # https://github.com/scaleway/scaleway-sdk-go/tree/master/scw#scaleway-config
 
+# This configuration file can be used with:
+# - scaleway-sdk-go (https://github.com/scaleway/scaleway-sdk-go)
```
Comment (QuentinBrosse) [inline scw/config.go]: What do you think about:
```
- Scaleway SDK Go (...)
- Scaleway CLI v2 (...)
- Scaleway Terraform Provider (...)
```

Comment (remyleone) [reply scw/config.go]: Fixed

File: scw/config.go [hunk 2]
```diff
@@ -22,6 +22,11 @@ const (
 const configFileTemplate = `# Scaleway configuration file
 # https://github.com/scaleway/scaleway-sdk-go/tree/master/scw#scaleway-config
 
+# This configuration file can be used with:
+# - Scaleway SDK Go (https://github.com/scaleway/scaleway-sdk-go)
+# - Scaleway CLI v2 (https://github.com/scaleway/scaleway-cli)
```
Comment (jerome-quere) [inline scw/config.go]: Scaleway CLI (>2.0.0) ?

Comment (remyleone) [reply scw/config.go]: Fixed

**Pull request merged**